Test/GherkinFeatureFiles/display-axis.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Create Axis for 3D Graph
+
+  This feature verifies the creation and manipulation of 3D axes in WebXr application.
+
+  Background:
+    Given the user has Oculus Quest 2
+    And the user has loaded the Quest Browser
+    And the user has loaded the WebXR application
+
+  @manualtesting
```

Comment:
In the future we dont need to add the tagging, that was my fault since the sample had tags on everything! Looks great though